src/components/fields/fields.component.tsx

```diff
@@ -84,6 +85,7 @@ export const Fields = memo(({ hideInput, color, onChange }: IFieldsProps) => {
               id="hex"
               className="rcp-field-input"
               value={fields.hex.value}
+              disabled={disabled}
```

Comment:
For inputs, it might also be a good idea to update the styles based on the `disabled` property.